wrangler-core/src/main/java/io/cdap/wrangler/utils/ColumnConverter.java

```diff
@@ -45,7 +47,7 @@ private ColumnConverter() {
    * @throws DirectiveExecutionException when a column matching the target name already exists
    */
   public static void rename(String directiveName, Row row, String column, String toName)
-    throws DirectiveExecutionException {
+      throws DirectiveExecutionException {
```

Comment:
nit: remove unchanged lines

Reply:
Had to make these changes due to checkstyle error: Line is longer than 120 charecters

Reply:
Alright